development/nautobot_config.py

```diff
@@ -97,6 +97,14 @@ def is_truthy(arg):
         "SSL": is_truthy(os.environ.get("REDIS_SSL", False)),
         "DEFAULT_TIMEOUT": 300,
     },
+    "custom_fields": {
+        "HOST": os.environ["REDIS_HOST"],
+        "PORT": int(os.environ.get("REDIS_PORT", 6379)),
+        "DB": 0,
+        "PASSWORD": os.environ["REDIS_PASSWORD"],
+        "SSL": is_truthy(os.environ.get("REDIS_SSL", False)),
+        "DEFAULT_TIMEOUT": 900,
+    },
```

Comment:
No `webhooks` queue here?

Also I just realized that these new queues also need to be added to `nautobot/docs/configuration/required-settings.md` as well.